eng/black-pyproject.toml

```diff
@@ -0,0 +1,10 @@
+[tool.black]
+line-length = 120
+extend-exclude = '''
+# Exclude some directories from formatting (note the slashes at surrounding the pattern)
+/(
+  | _vendor
+  | _generated
```

Comment:
Is it possible to exclude the project specific directories?

Reply:
I'm unsure what the Azure SDK team's stance on that would be.

What did you have in mind specifically to exclude?

Reply:
Removed the exclusions for `_vendor` and `_generated`, so we're probably good here.

Reply:
Resolved by [2b42d02](https://github.com/Azure/azure-sdk-for-python/pull/28835/commits/2b42d02000544906f5d39ea97c27421c8023f748)